packages/frontend/src/components/FilterBar/dateInfo.ts

```diff
@@ -15,9 +15,10 @@ import {
   startOfYear,
   startOfYesterday,
 } from 'date-fns';
-import { Locale, nb } from 'date-fns/locale';
+import { fromZonedTime } from 'date-fns-tz';
+import { type Locale, nb } from 'date-fns/locale';
 import { t } from 'i18next';
-import { InboxItemInput } from '../../pages/Inbox/Inbox.tsx';
+import type { InboxItemInput } from '../../pages/Inbox/Inbox.tsx';
```

Comment:
Kinda weird that a component in the `src/components` folder import a type from the `src/pages` folder. Seems like it maybe should be the other way around? Or at the very least that the type should simply be dublicated.

Reply:
Enig, egentlig holder det at den til dels confirmer til et interface for å være filtrerbar, så skriver om.